go/obscuronode/enclave/enclave.go

```diff
@@ -47,11 +47,20 @@ type Enclave interface {
 	RoundWinner(parent common3.L2RootHash) (common2.ExtRollup, bool)
 	Stop()
 
-	// TestPeekHead - only available for testing purposes
-	TestPeekHead() BlockState
+	// L1Height returns the current L1 height
+	L1Height() int
 
-	// TestDb - only available for testing purposes
-	TestDB() DB
+	// L1HeightHash returns the current L1 height Hash
+	L1HeightHash() common3.L1RootHash
+
+	// L2Height returns the current L2 height
+	L2Height() int
```

Comment:
I'd say this is also something to keep track on the unencrypted "node" code.
It can be calculated based on the plaintext headers alone

Reply:
Removed from the enclave 👍